lighthouse-core/report/html/report-styles.css

```diff
@@ -930,14 +919,16 @@ summary.lh-passed-audits-summary {
 .lh-table-column--url {
   min-width: 250px;
   white-space: nowrap;
-  max-width: 0;
 }
 
-/* Keep bytes columns narrow if they follow the URL column */
+/* Keep columns narrow if they follow the URL column */
+/* 12% width was found to be a good width for en and i18n without being too wide */
 .lh-table-column--url + th.lh-table-column--bytes,
 .lh-table-column--url + .lh-table-column--bytes + th.lh-table-column--bytes,
+.lh-table-column--url + .lh-table-column--ms, 
+.lh-table-column--url + .lh-table-column--ms + th.lh-table-column--bytes,
 .lh-table-column--url + .lh-table-column--bytes + th.lh-table-column--timespanMs {
-  max-width: var(--bytes-col-width);
+  width: 12%;
 }
 
 .lh-table-column--code {
```

Comment:
while you're here, i notice that this var doesnt exist anymore.

it's used on these two audits at least

![image](https://user-images.githubusercontent.com/39191/46379165-d6e8d080-c652-11e8-8f34-c5afb8e59212.png)


seems like its fine without a max-width being set.